docs/guide/appearances.md

```diff
@@ -11,8 +11,6 @@ An asphalt material. The color can be overridden using the `colorOverride` field
 
 %end
 
-Derived from [PBRAppearance](../reference/pbrappearance.md).
-
```

Comment:
These changes in this file seems wrong.
Given that this file is not related to your changes, could you please revert it to the original version?
We will later inspect why the script didn't work correctly.